noodles-editor/src/noodles/components/menu.tsx

```diff
@@ -494,8 +498,10 @@ export function NoodlesMenubar({
       addToRecentProjects(projectName)
       // Update store with directory handle returned from save
       setCurrentDirectory(result.data.directoryHandle, projectName)
+      analytics.track('project_saved', { storageType, isFirstSave: false })
```

Comment:
Is there a `isFirstSave: true`? Edit: I guess there actually isn't right now until we have more project management